modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -0,0 +1,45 @@
+= Node-wise Partition Recovery
+:description: Feature to recover dead nodes.
+
+Multi-node or entire-AZ failures (especially in cloud environments) and some forms of human error may result in ‘stuck’ partitions with fewer replicas than required to make a quorum. In these scenarios, some data loss is unavoidable but admins need to recover what they can and move on.
+
+IMPORTANT: This is a last ditch measure when all other recovery options have failed. In some cases, no remaining replicas may exist for the partitions on the dead nodes. This recovery method is used in scenarios where you are already experiencing data loss - the goal is to stop the loss of any more data.
+
+== Prerequisites
+
+This feature has two prerequisites for recovering partitions:
+
+* Your cluster must have one or more `dead` nodes.
+* One or more partitions on these nodes must have no majority.
+
+Executing a node-wise partition recovery with no nodes or partitions meetings these criteria will result in Redpanda performing no action.
+
+== Performing the recovery operation
+
+Node-wise partition recovery is performed via the RPK command `rpk cluster partitions unsafe-recover`. This command includes an interactive prompt to confirm execution of the generated recovery plan as it is a destructive operation. When you trigger node-wise partition recovery, the partitions on the node are rebuilt in a best-effort basis. You may lose data as a result of executing this operation.
```

Comment:
lose 'some data which was not yet replicated to the surviving partition replicas as a result of executing this operation'

besides that this looks good.